### PR TITLE
Rename binaries according to new rules (QC-164)

### PR DESCRIPTION
**Pull request by Barthelemy**

### CODE REVIEW

**Comment by Barthelemy** [comment]
I tried to update all references, including in the doc. Old names are still available via symlinks, just to avoid a surge in user support. It should be advertised to the users in a dedicated email.

File: Framework/CMakeLists.txt [hunk 1]
```diff
@@ -139,29 +139,53 @@ set(
...
+  o2-qc-run-readout
+  o2-qc-run-merger-test
+  o2-qc-run-readout-for-data-dump
+  o2-qc-repository-benchmark
 )
 
+# These were the original names before the convention changed.
+# We will get rid of them but for the time being we want to create symlinks to avoid confusion.
+set(
+        EXE_OLD_NAMES
```
Comment (knopers8) [inline Framework/CMakeLists.txt]: Could you modify it to have the two-space indentation, as in the rest of the file?

**Comment by Barthelemy** [comment]
agreed and fixed !

**Comment by knopers8** [comment]
Thank you!

**Pull request merged**